client/packages/common/src/ui/components/inputs/Upload/Upload.tsx

```diff
@@ -17,12 +17,14 @@ export interface UploadProps {
   color?: 'primary' | 'secondary' | 'gray';
   maxSize?: number;
   onUpload: <T extends File>(files: T[]) => void;
+  customWidth?: string;
 }
 
 export const Upload: FC<UploadProps> = ({
   accept,
   color = 'secondary',
   onUpload,
+  customWidth,
```

Comment:
This is to allow side by side placement of buttons in mobile view - but is a little bit janky. Would welcome better ideas for how to structure mobile specific css where the upload button is generic to many parts of the app.